vaccine_feed_ingest_schema/location.py

```diff
@@ -50,10 +181,10 @@ class Contact(BaseModel):
     }
     """
 
-    contact_type: Optional[str]
-    phone: Optional[str]
-    website: Optional[str]
-    email: Optional[str]
+    contact_type: Optional[ContactType]
+    phone: Optional[str] = Field(regex=US_PHONE_RE.pattern)
+    website: Optional[HttpUrl]
+    email: Optional[EmailStr]
     other: Optional[str]
```

Comment:
A common PR comment on normalizers is asking them to split from one `Contact` with multiple attributes filled out to an array of multiple `Contact`s

Is there a straightforward way to enforce that only one of these fields is set?

Reply:
I can look into multi-field @validators. I would like to get this PR landed though, and we can wait to cut a release until we do that.

Reply:
Sounds good!